lifecycle/lifecycle/auth/authenticate.py

```diff
@@ -46,7 +46,7 @@ def read_auth_token_header(request: Request) -> Optional[str]:
     if not auth_token:
         auth_token = request.headers.get("X-Racetrack-User-Auth")
     if not auth_token:
-        auth_token = request.headers.get("X-Racetrack-Fatman-Auth")
+        auth_token = request.headers.get("X-Racetrack-Job-Auth")
```

Comment:
Please revert this line as it was supposed to keep backwards compatibility with the old header name